tests/factories/cht/generate.js

```diff
@@ -51,7 +60,75 @@ const createData = ({ healthCenter, user, nbrClinics=10, nbrPersons=10 }) => {
   return { clinics, reports, persons };
 };
 
-const createHierarchy = ({ name, user=false, nbrClinics=50, nbrPersons=10 }) => {
+const createClinic = (index, healthCenter) => {
+  const firstName = FIRST_NAMES[index % FIRST_NAMES.length];
+  const lastName = LAST_NAMES[index % LAST_NAMES.length];
+  const personName = `${firstName} ${lastName}`;
+  const personPhoneNumber = PHONE_NUMBERS[index % PHONE_NUMBERS.length];
+
+  const primaryContact = personFactory.build({
+    name: personName,
+    phone: personPhoneNumber
+  });
+
+  const clinic = placeFactory.place().build({
+    type: 'clinic',
+    parent: { _id: healthCenter._id, parent: healthCenter.parent },
+    name: `${personName} Family`,
+    contact: primaryContact
+  });
+
+  primaryContact.parent = { _id: clinic._id, parent: clinic.parent };
+
+  return { clinic, primaryContact };
+};
+
+const createAdditionalPersons = (nbrPersons, clinic) => {
+  return Array.from({ length: nbrPersons - 1 }).map((_, i) => {
```

Comment:
Code style: [One method per line should be used](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#method-chaining)
```suggestion
  return Array
    .from({ length: nbrPersons - 1 })
    .map((_, i) => {
```